<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Microsoft JhengHei UI</family>
    </font>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>211</width>
      <height>661</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Arm Control Panel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QGroupBox" name="groupBox1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="title">
      <string>Joint 1</string>
     </property>
     <widget class="QPushButton" name="pushButtonCCW1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CCW</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCW1">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CW</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickInterval">
       <number>1</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>191</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Joint 2</string>
     </property>
     <widget class="QPushButton" name="pushButtonCCW2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CCW</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCW2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CW</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickInterval">
       <number>1</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>191</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Joint 3</string>
     </property>
     <widget class="QPushButton" name="pushButtonCCW3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CCW</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCW3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CW</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickInterval">
       <number>1</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>191</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Paint Brush</string>
     </property>
     <widget class="QPushButton" name="pushButtonPaint">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Paint</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonUndo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Undo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonClear">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Paint Continuously</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>191</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Paint Brush Color</string>
     </property>
     <widget class="QPushButton" name="redButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Red</string>
      </property>
     </widget>
     <widget class="QPushButton" name="greenButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Green</string>
      </property>
     </widget>
     <widget class="QPushButton" name="blueButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Blue</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="redEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>255</string>
      </property>
      <property name="maxLength">
       <number>255</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="greenEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>255</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="blueEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>255</string>
      </property>
     </widget>
     <widget class="QPushButton" name="customButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Custom</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox4_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>191</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Brush Position Control</string>
     </property>
     <widget class="QPushButton" name="pushButtonIncrX">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>+X (D)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonIncrY">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>+Y (W)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDecrX">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>50</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>-X (A)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDecrY">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>-Y (S)</string>
      </property>
     </widget>
    </widget>
    <zorder>groupBox5</zorder>
    <zorder>groupBox1</zorder>
    <zorder>groupBox2</zorder>
    <zorder>groupBox3</zorder>
    <zorder>groupBox4</zorder>
    <zorder>groupBox4_2</zorder>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>661</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1004</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
